docs/language/learn-ql/csharp/introduce-libraries-csharp.rst

```diff
@@ -1,18 +1,20 @@
-Introducing the CodeQL libraries for C#
-=======================================
+CodeQL library for C#
+=====================
 
-Overview
---------
+When you need to analyze a C# program, you can make use of the large collection of classes in the C# library for CodeQL.
```

Comment:
I *think* this needs to be "CodeQL library for C#" because the library is written in CodeQL.